integration/src/shopify-token-exchange/shopify-token-exchange.module.ts

```diff
@@ -0,0 +1,28 @@
+import { ShopifyAuthModule } from '@nestjs-shopify/auth';
+import { Module } from '@nestjs/common';
+import { MyAfterAuthHandler } from './my-after-auth-handler';
+import { MyAuthModule } from './my-auth.module';
+
+@Module({
+  imports: [
+    ShopifyAuthModule.forRootAsyncOnline({
+      imports: [MyAuthModule],
+      authStrategy: 'TOKEN_EXCHANGE',
+      useFactory: (myAuthHandler: MyAfterAuthHandler) => ({
+        returnHeaders: true,
+        afterAuthHandler: myAuthHandler,
+      }),
+      inject: [MyAfterAuthHandler],
+    }),
```

Comment:
I would expect the `authStrategy` to only be part of the `ShopifyAuthModuleOptions` interface. Not in the `ShopifyAuthModuleAsyncOptions` interface.

```suggestion
    ShopifyAuthModule.forRootAsyncOnline({
      imports: [MyAuthModule],
      useFactory: (myAuthHandler: MyAfterAuthHandler) => ({
        authStrategy: 'TOKEN_EXCHANGE',
        returnHeaders: true,
        afterAuthHandler: myAuthHandler,
      }),
      inject: [MyAfterAuthHandler],
    }),
```